lib/kickster/generator.rb

```diff
@@ -1,7 +1,7 @@
+require 'fileutils'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.